docs/guides/high-availability/concepts-memory-and-cpu-sizing.adoc

```diff
@@ -29,83 +29,84 @@ Summary:
 
 Recommendations:
 
-* The base memory usage for an inactive Pod is 1000 MB of RAM.
-
-* For each 100,000 active user sessions, add 500 MB per Pod in a three-node cluster (tested with up to 200,000 sessions).
-+
-This assumes that each user connects to only one client.
-Memory requirements increase with the number of client sessions per user session (not tested yet).
+* The base memory usage for a Pod including caches of Realm data and 10,000 cached sessions is 1250 MB of RAM.
 
 * In containers, Keycloak allocates 70% of the memory limit for heap based memory. It will also use approximately 300 MB of non-heap-based memory.
 To calculate the requested memory, use the calculation above. As memory limit, subtract the non-heap memory from the value above and divide the result by 0.7.
 
-* For each 45 password-based user logins per second, 1 vCPU per Pod in a three-node cluster (tested with up to 300 per second).
+* For each 15 password-based user logins per second, allocate 1 vCPU to the cluster (tested with up to 300 per second).
```

Comment:
Would this then compute to, 135 user logins per second, if only the user login scenario is load tested against the 3 pod cluster in an Active/Active setup ? @ahus1

Reply:
In a 2x3 Pods A/A setup and 135 logins per seconds, you would see 1.5 vCPUs used on each pod when both clusters are active. If then one site goes down, you would then see the usage of 3.0 vCPUs on each of the three remaining active Pods.

I'd think you have a choice here then either request 1.5 vCPUs per Pod and rely on the cluster to have enough bandwidth when one site goes down, or request 3.0 vCPUs per Pod to be on the safe side. Maybe request 1.5 vCPUs and then have an auto-scaler scale up 6 Pod in the remaining DC if you can afford the delay of an autoscaler on the pod level and the cluster level. IMHO there is no simple answer to that. I'll see if I can provide an answer on this, still.

Reply:
Thank you Alex, this helps!